tests/meta/orders/test_meta_orders_status.py

```diff
@@ -697,3 +700,47 @@ def test_store_existing_single_sample_from_trio(
     assert len(new_families[0].links) == 1
     assert not new_families[0].links[0].mother
     assert not new_families[0].links[0].father
+
+
+def test_store_rerun_case(
+    orders_api: OrdersAPI, base_store: Store, mip_status_data: dict, ticket: str
+):
+    # GIVEN a basic store with no samples or nothing in it + scout order
+    assert base_store.samples().first() is None
+    assert base_store.families().first() is None
+
+    submitter: MipDnaSubmitter = MipDnaSubmitter(lims=orders_api.lims, status=orders_api.status)
+
+    # WHEN storing the order
+    submitter.store_items_in_status(
+        customer=mip_status_data["customer"],
+        order=mip_status_data["order"],
+        ordered=dt.datetime.now(),
+        ticket=ticket,
+        items=mip_status_data["families"],
+    )
+
+    base_store.close()
+    new_cases = base_store.families().all()
+
+    # Save internal id
+    stored_cases_internal_ids = dict([(case["name"], case["internal_id"]) for case in new_cases])
+    for case in mip_status_data["families"]:
+        case["internal_id"] = stored_cases_internal_ids[case["name"]]
+
+    new_ticket = ticket
```

Comment:
Not needed anymore right?

Reply:
Then new_ticket variable I don't but I do need to store the internal id of the previous cases